library/RAVE/src/RAVE/eye_tracker/EyeTrackerModel.py

```diff
@@ -20,6 +20,7 @@ def __init__(self):
         head is only useful / used during the training process.
         """
         super(EyeTrackerModel, self).__init__()
+        torch.hub._validate_not_a_forked_repo=lambda a, b, c: True
```

Comment:
Qu'est-ce que ça fait ça / pourquoi c'est nécessaire ?

Reply:
Fix un erreur quand on telecharge le repo, https://stackoverflow.com/a/69072492

Reply:
Ok, looks good to me alors !